catalyst-toolbox-lib/src/recovery/tally.rs

```diff
@@ -188,20 +187,20 @@ pub fn recover_ledger_from_logs(
                 let block_date = fragment_log_timestamp_to_blockdate(time, &timeframe, &ledger)
                     .expect("BlockDates should always be valid for logs timestamps");
 
-                println!("Fragment processed {}", fragment.hash());
+                dbg!("Fragment processed {}", fragment.hash());
```

Comment:
This would be good to put behind a `--verbose` command line flag. Not necessary for this PR, though.

Reply:
I like that idea. Probably to be added later as you suggest.